index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>my-project</title>
+  </head>
+  <body>
+    <div id="app"></div>
```

Comment:
I strongly recommend you avoid using ids unless you need to. This should just be a unique classname